lib/lit-element.d.ts

```diff
@@ -0,0 +1,21 @@
+import { TemplateResult } from '../node_modules/lit-html/lit-html.js';
```

Comment:
I don't think these files should be checked in - they are generated

Reply:
we need to for npm right?